### DISCUSSION TITLE
Best practices for upgrading Hydra to version v2

### DISCUSSION THREAD

**Discussion by oninugarte** [ory/hydra#3532] [Q&A]
Hello Hydra people!
Currently in my company we are working with hydra and now we are in the situation to do a version upgrade from v1.11.8 to the latest v2.1.2. I would like to get an opinion on the best way to do this upgrade since in between we have breaking changes among other changes and I would like to optimize the process. Maybe someone here has been in the same situation and can give me some advice.

**Comment by vinckr** [comment]
Hello @oninugarte 
Have a look at the [Apply Upgrades](https://www.ory.sh/professional-services/) documentation.
Generally it depends a lot on how you use Ory Hydra in production.

By the way, Ory is now offering support for self-hosted Ory Hydra, have a look here if you are interested: https://www.ory.sh/professional-services/
